common/src/main/java/org/dash/wallet/common/data/WalletUIConfig.kt

```diff
@@ -63,6 +63,8 @@ open class WalletUIConfig @Inject constructor(
         val SHOW_TAP_TO_HIDE_HINT = booleanPreferencesKey("show_tap_to_hide_balance_hint")
         val SELECTED_CURRENCY = stringPreferencesKey("exchange_currency")
         val EXCHANGE_CURRENCY_DETECTED = booleanPreferencesKey("exchange_currency_detected")
+        val LAST_TOTAL_BALANCE = longPreferencesKey("last_total_balance")
+        val LAST_MIXED_BALANCE = longPreferencesKey("last_mixed_balance")
```

Comment:
Save the last calculated values so that they can be emitted first when the app starts